src/qmflows/packages/packages.py

```diff
@@ -286,6 +291,15 @@ def __init__(self, pkg_name: str) -> None:
         """
         self.pkg_name = pkg_name
 
+        # Ensure compatibility with the (typing-only) `builtins.function` class
```

Comment:
@BvB93  can you please give an example about when are these `__name__`, `__qualname__` and `__annotations__` necessary?

Reply:
It's mostly relevant when working on function the objects themselves (in Auto-FOX or assertionlib for example).
Above-mentioned attributes are always present for proper functions (_i.e._ `types.FunctionType` instances) but this is, annoyingly enough, not necessarily the case for all callables.